workflow.md

```diff
@@ -100,8 +100,8 @@ The current `instance` is accessed in the same way as the `context`.
 import pyblish.api
 
 class MyPlugin(pyblish.api.Plugin):
-    def process(self, context):
-        self.log.info("Publishing from {cwd}".format(**context.data))

```

Comment:
I thought it was awkard that this exact example was used twice on that page. Are you sure you're not mistaking that this was the example for the `context`? I can understand you might want to show again how the context was handled. I just think it's odd it's not showing the how the instance can be added to `process()` with the Dependency Injection.

Maybe something like this would be best:

```python
import pyblish.api

class MyPlugin(pyblish.api.Plugin):
    def process(self, context, instance):
        self.log.info("Publishing from {cwd}".format(**context.data))
        self.log.info("Publishing instance with its own data {0}".format(instance.data))
```

This could also show that `instance.data != context.data`.